custom/up_nrhm/tests/__init__.py

```diff
@@ -45,11 +43,8 @@ def setUpModule():
     _call_center_domain_mock.stop()
 
 
+@db_tests_only
 def tearDownModule():
-    if isinstance(Domain.get_db(), Mock):
-        # needed to skip setUp for javascript tests thread on Travis
-        return
-
     _call_center_domain_mock = mock.patch(
```

Comment:
I also notice that the rest of these setup/teardown functions look suspiciously similar.  If I look too long at this I'd probably get sucked in to a refactor

:see_no_evil: